FestivalPlanner/films/views.py

```diff
@@ -187,23 +187,41 @@ class FilmsListView(LoginRequiredMixin, ListView):
     template_name = FilmsView.template_name
     context_object_name = 'film_rows'
     http_method_names = ['get']
+    object_list = None
     title = 'Film Rating List'
     class_tag = 'rating'
     highest_rating = FilmFanFilmRating.Rating.values[-1]
     eligible_ratings = FilmFanFilmRating.Rating.values[LOWEST_PLANNABLE_RATING:]
     short_threshold = timedelta(minutes=MAX_SHORT_MINUTES)
     fan_list = get_present_fans()
-    section_list = Section.sections.all()
-    subsection_list = Subsection.subsections.all()
+    section_list = None
+    subsection_list = None
```

Comment:
Please remove these initializations as to make the lists object attributes, not class attributes.

Reply:
Done.